client/state/data-getters/index.js

```diff
@@ -23,6 +23,31 @@ import {
 import { convertToSnakeCase } from 'state/data-layer/utils';
 import { dummyTaxRate } from 'lib/tax'; // #tax-on-checkout-placeholder
 
+/**
+ * Fetches content from a URL with a GET request
+ *
+ * The ID here is obscured and so **this should not
+ * be used inside a React component** but rather it
+ * should only be used in something like `waitForData()`
+ * where the wrapper defines the shape of the output
+ *
+ * @example
+ * waitForData( {
+ *     planets: getAtURL( 'https://swapi.co/api/planets/' ),
```

Comment:
Should be `planets: () => getAtURL( 'https://swapi.co/api/planets/' )` I think

Reply:
good catch! updated in cbd1788